src/main/java/entities/Quiz.java

```diff
@@ -16,4 +21,25 @@ public Quiz(String quizName, List<Deck> decks) {
         this.quizName = quizName;
         this.decks = decks;
     }
+
+    public List<Deck> getDecks(){
+        return this.decks;
+    }
+
+    public String getQuizName(){
+        return this.quizName;
+    }
+
+    public static Quiz getQuiz(String quizId){
```

Comment:
These methods were created by Jasmine, just be sure we only use one when we merge these changes onto the main branch